Solutions/CohesitySecurity/SolutionMetadata.json

```diff
@@ -0,0 +1,15 @@
+{
+    "publisherId": "cohesity",
+    "offerId": "cohesity_sentinel_data_connector",
+    "firstPublishDate": "2022-10-10",
+    "providers": ["Cohesity"],
+    "categories": {
+        "domains": ["DevOps"]
+    },
+    "support": {
+        "name": "Cohesity Corporation",
+        "email": "cohesity_support_email",
+        "tier": "Microsoft",
```

Comment:
Hey @yinghuang123, can you please change tier to "Partner" as the support information is of Cohesity, thanks.

Reply:
Done